stub.php

```diff
@@ -1,4 +1,4 @@
-<?php
+<?php declare(strict_types=1);
```

Comment:
this file is supposed to work on PHP5, I don't think we can use this here?

Reply:
It only prints a warning `Warning: Unsupported declare 'strict_types'` but I agree, the stub file is a special case. I first tried to include it in the cs-fixer but in the end I removed it and tried to only keep the sane changes.

Reply:
Removed in 9e4c16d5ca25e07c74e3cc8035f27e21f52f2b4c